guides/source/i18n.md

```diff
@@ -1208,7 +1208,7 @@ Resources
 
 * [Google group: rails-i18n](https://groups.google.com/g/rails-i18n) - The project's mailing list.
```

Comment:
This looks dead, maybe axe it?

Reply:
Yes, good point.
Maybe remove the authors on line 1214/1218 as well?
I don't think we mention authors anywhere else?

Reply:
I don't have a solid case, but my guess is that "Authors" here is actually who originally wrote this guide (it looks like it came from `workingwithrails.com` many years ago).

Reply:
Thanks @carlosantoniodasilva . I've added some comments to the TODO.